openstack/pipeline/02configure

```diff
@@ -28,6 +28,11 @@ else
     MOD_PARAMS[__NUM_NEUTRON_GATEWAY_UNITS__]=0
 fi
 
+# Default to OVN for > Train
+if has_min_release ussuri && ! has_opt --ovs; then
+    set -- $@ --ovn && cache $@
```

Comment:
nit: --ml2-ovn ?

Also, the commit message doesn't mention anything about adding the option, or the new --ml2-ovs option either.

Reply:
Changed. I will update the commit message.